lib/osf-components/addon/components/file-browser/file-item/template.hbs

```diff
@@ -26,6 +29,17 @@
                 {{moment-format @item.dateModified 'YYYY-MM-DD hh:mm A'}}
             </span>
         {{/if}}
-        <FileActionsMenu @item={{@item}} />
+        <FileActionsMenu
+            @item={{@item}}
+            @toggleRenameModal={{this.openRenameModal}}
+            local-class='FileActionsButton'
+        />
+        <FileBrowser::FileRenameModal
+            @isOpen={{this.renameModalOpen}}
+            @onClose={{action this.closeRenameModal}}
+            @item={{@item}}
+            @manager={{@manager}}
+            @disabled={{this.disabled}}
+        />
```

Comment:
FileRenameModal will need to be invoked on whatever element you would like the rename functionality to be on. Currently this also requires adding a tracked property of disabled and renameModalOpen. I have tested repositioning the element, but it does not work the same. Would you like to discuss other ways of designing the component? It could require passing another set of parameters and would be open to changing the code.

Reply:
FileRenameModal is now invoked on FilesActionMenu.